easybuild/easyconfigs/s/StringTie/StringTie-1.3.5-foss-2019a.eb

```diff
@@ -0,0 +1,30 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+easyblock = 'MakeCp'
+
+name = 'StringTie'
+version = '1.3.5'
+
+homepage = 'http://ccb.jhu.edu/software/%(namelower)/'
+description = "StringTie is a fast and highly efficient assembler of RNA-Seq alignments into potential transcripts."
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+
+source_urls = ['http://ccb.jhu.edu/software/%(namelower)s/dl/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['f16ec07d4869b7656bc1a9b6eb662fd9404c122d18431c845e93d810840c5db9']
+
+dependencies = [
+    ('zlib', '1.2.11'),
+]
+
+# release option must be given to 'make' otherwise a DEBUG version is built by default
+buildopts = " release"
```

Comment:
@arielzn We don't have this (or the `zlib` dep) in `StringTie-1.3.5-foss-2018b.eb`, is it worth fixing that there?

Reply:
the `buildopts = release` i would say yes, by running just `make` the code is built with debug symbols and forces `-O0` 

You can verify in some build logs with the upstream version, that it's dumped something like `WARNING: built DEBUG version [much slower], use "make clean release" for a faster, optimized version of the program.`

The missing zlib dep, gave me issues when trying to build in a CentOS 6 with a really old system zlib. Actually this made me look at the logs and notice the other message at some point.

Reply:
it's also specified in the [README](https://github.com/gpertea/stringtie/blob/master/README.md)

Reply:
fixed in existing StringTie 1.3.5 easyconfig via #7845